src/main/java/com/conveyal/datatools/manager/auth/Auth0Users.java

```diff
@@ -78,7 +92,7 @@ private static String doRequest(URI uri) {
         HttpClient client = HttpClientBuilder.create().build();
         HttpGet request = new HttpGet(uri);
 
-        request.addHeader("Authorization", "Bearer " + AUTH0_API_TOKEN);
+        request.addHeader("Authorization", "Bearer " + getApiToken());
```

Comment:
when the return of `getApiToken()` is null this whole request will still continue when it actually should probably immediately stopped since there isn't a valid API token.